ga4gh/_protocol_definitions.py

```diff
@@ -207,6 +207,52 @@ def __init__(self, **kwargs):
         """
 
 
+class Attributes(ProtocolElement):
+    """
+    Type defining a collection of attributes associated with various
+    protocol   records.  Each attribute is a name that maps to an
+    array of one or more   values.  Values can be strings, external
+    identifiers, or ontology terms.   Values should be split into the
```

Comment:
This isn't the biggest deal in the world, but it seems like there are some weird double-spaces in these comments.  Is `process_schemas.py` inserting extra spaces?